runtime/lua/vim/_meta/vimfn.lua

```diff
@@ -47,7 +47,7 @@ function vim.fn.acos(expr) end
 ---
 --- @param object any
 --- @param expr any
---- @return any
+--- @return any # Resulting |List| or |Blob|, or 1 if {object} is not a |List| or a |Blob|.
```

Comment:
`returns_desc` field defined in eval.lua. Now just need AI to generate all those `returns_desc` :)